internal/shared/client_test.go

```diff
@@ -0,0 +1,114 @@
+package shared
+
+import (
+	"fmt"
+	"github.com/broadinstitute/revere/internal/configuration"
+	"github.com/go-resty/resty/v2"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestBaseClient(t *testing.T) {
```

Comment:
Same thing here. Are we just re-testing a 3rd party library here?

Reply:
More testing that we are configuring it the way that config.go says we will. I think this is a test we can absolutely toss if this "base client" thing doesn't get any more complicated--just not sure we'll know till a bit more is built out.